diff --git a/lapce-app/src/panel/data.rs b/lapce-app/src/panel/data.rs
--- lapce-app/src/panel/data.rs
+++ lapce-app/src/panel/data.rs
@@ -22,19 +22,27 @@
     let mut order = PanelOrder::new();
     order.insert(
         PanelPosition::LeftTop,
         im::vector![
+            PanelKind::Plugin,
+        ],
+    );
+    order.insert(
+        PanelPosition::RightTop,
+        im::vector![
             PanelKind::FileExplorer,
-            PanelKind::Plugin,
             PanelKind::SourceControl,
             PanelKind::Debug,
         ],
     );
     order.insert(
-        PanelPosition::BottomLeft,
-        im::vector![PanelKind::Terminal, PanelKind::Search, PanelKind::Problem,],
-    );
-
+        PanelPosition::BottomRight,
+        im::vector![
+            PanelKind::Terminal,
+            PanelKind::Search,
+            PanelKind::Problem,
+        ],
+    );
     order
 }

 #[derive(Clone, Serialize, Deserialize)]
@@ -76,9 +84,9 @@
         styles.insert(
             PanelPosition::LeftTop,
             PanelStyle {
                 active: 0,
-                shown: true,
+                shown: false,
                 maximized: false,
             },
         );
         styles.insert(
@@ -92,9 +100,9 @@
         styles.insert(
             PanelPosition::BottomLeft,
             PanelStyle {
                 active: 0,
-                shown: true,
+                shown: false,
                 maximized: false,
             },
         );
         styles.insert(
@@ -108,9 +116,9 @@
         styles.insert(
             PanelPosition::RightTop,
             PanelStyle {
                 active: 0,
-                shown: false,
+                shown: true,
                 maximized: false,
             },
         );
         styles.insert(
